Automattic/sensei
# If sensei pro active, add groups menu item

Fixes #1385
(This PR is in coordination with PR on sensei PRO: https://github.com/Automattic/sensei-pro/pull/1401)

### Changes proposed in this Pull Request
- Add groups menu items to the proper location in the menu 
<img width="231" alt="image" src="https://user-images.githubusercontent.com/7208249/177147998-d6b78fbc-67ba-4dd2-9935-4b0797b19a20.png">

### Testing instructions

1. When sensei PRO is installed
- Make sure in Sensei PRO pr is merged OR use that branch : https://github.com/Automattic/sensei-pro/pull/1401
- Open Admin view
- Make sure Groups menu item is in the proper location
- When clicking it http://sensei.local/wp-admin/edit.php?post_type=group page should be open

1. When sensei PRO is NOT installed
- Make sure in Sensei PRO pr is merged OR use that branch 
- Open Admin view
- Make sure Groups menu item is on the proper location
- When clicking it marketing landing page should be open: 
<img width="932" alt="image" src="https://user-images.githubusercontent.com/7208249/177148382-74f0afea-096a-4f82-91d4-5a8ff5ab8985.png">
